app/pages/settings/helpers/getBearerToken.js

```diff
@@ -0,0 +1,21 @@
+export default async function getBearerToken (authClient) {
+  let token = ''
+  if (authClient) {
+    try {
+      const authToken = await authClient.checkBearerToken()
+      // API method between PJC's auth and oauth clients are inconsistent.
+      // First party auth client returns a string of the bearer token
+      // Oauth client returns an object of the bearer token, expiration, and token type
```

Comment:
The PFE user settings page doesn’t use third party OAuth.

If you’re logged in, you can always get your bearer token from `auth.checkBearerToken()`. 

See https://zooniverse.github.io/panoptes-javascript-client/#panoptes-javascript-client-auth